samples/cpp/tutorial_code/ImgProc/Threshold_inRange.cpp

```diff
@@ -60,17 +60,17 @@ int main(int argc, char* argv[])
     //! [cap]
 
     //! [window]
-    namedWindow(window_capture_name);
-    namedWindow(window_detection_name);
+    namedWindow(window_capture_name,WINDOW_NORMAL);
+    namedWindow(window_detection_name,WINDOW_NORMAL);
     //! [window]
 
     //! [trackbar]
     // Trackbars to set thresholds for HSV values
     createTrackbar("Low H", window_detection_name, &low_H, max_value_H, on_low_H_thresh_trackbar);
-    createTrackbar("High H", window_detection_name, &high_H, max_value_H, on_high_H_thresh_trackbar);
     createTrackbar("Low S", window_detection_name, &low_S, max_value, on_low_S_thresh_trackbar);
-    createTrackbar("High S", window_detection_name, &high_S, max_value, on_high_S_thresh_trackbar);
     createTrackbar("Low V", window_detection_name, &low_V, max_value, on_low_V_thresh_trackbar);
+    createTrackbar("High H", window_detection_name, &high_H, max_value_H, on_high_H_thresh_trackbar);
+    createTrackbar("High S", window_detection_name, &high_S, max_value, on_high_S_thresh_trackbar);
```

Comment:
There is no strict order of created trackbars.
QT backend may differ from others (GTK is default on Linux, Win32 on Windows).
Need to update highui module first instead and after that change samples.